### ISSUE TITLE
Importing psafe3

### ISSUE THREAD

**Issue by bgaluszka** [bwesterb/pol#10]
When trying to import psafe3 file I'm getting

```
An unhandled exception occured:

   Traceback (most recent call last):
     File "/usr/local/lib/python2.7/dist-packages/pol/main.py", line 1273, in _run_command
       return self.args.func()
     File "/usr/local/lib/python2.7/dist-packages/pol/main.py", line 1158, in cmd_import_psafe3
       header, records = pol.importers.psafe3.load(f, ps3pwd)
     File "/usr/local/lib/python2.7/dist-packages/pol/importers/psafe3.py", line 97, in load
       struct.unpack("<I", d)[0])
   error: unpack requires a string argument of length 4
```

**Comment by bwesterb** [comment]
Thanks for reporting the issue.  Could you try to create a psafe3 password safe (without containing your actual passwords) which also triggers this error?  Then you can send it to me.

**Comment by bwesterb** [comment]
Thank you very much for reporting the issue and the test-case you sent.  This issue should be fixed in the commit above which is in 0.3.5.

**Comment by bgaluszka** [comment]
Not a problem, and thank you for your software and a fix :)
